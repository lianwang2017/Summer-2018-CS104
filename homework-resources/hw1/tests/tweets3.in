@user1, @user2 and @user3 we are #hashtag #to #extreme

#hashtag #to #extreme @user1 is important @user1     because     #extreme

@newuser go for it #JustDoIt in a Rainbow

#HelloWorld

@ev

@twitter		tabbed   spaced #tweet
