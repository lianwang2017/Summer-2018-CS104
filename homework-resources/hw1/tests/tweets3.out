1. Number of tweets=6
2. Unique users
user1,
user2
user3
user1
newuser
ev
twitter
3. Unique hashtags
hashtag
to
extreme
JustDoIt
HelloWorld
tweet
